e2e/tests/content-type-builder/ctb.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { navToHeader } from '../../utils/shared';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+
+test.describe('Edit View CTB', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test('A user should be able to navigate to the Edit View of the content type builder and see the contentype fields', async ({
+    page,
+  }) => {
+    await navToHeader(page, ['Content-Type Builder'], 'Article');
+    const modal = page.getByRole('button', { name: 'Close' });
+    if (modal.isVisible()) {
+      modal.click();
+    }
+    await page.getByRole('link', { name: 'Shop' }).click();
+    await expect(
+      page.getByRole('button', {
+        name: 'Add another field to this component',
+      })
+    ).toBeVisible();
+    await expect(page.getByRole('button', { name: 'Add a component' })).toBeVisible();
+    await expect(
+      page.getByRole('button', { name: 'Add another field to this single type' })
+    ).toBeVisible();
```

Comment:
Can you look at the other e2e tests and see how most people have written comments to explain what's ging on?

it's not quite clear at the moment which might make it harder to maintain in the long run :)